Remove User API impl. completed

Swanandm1994 opened the pull request:
@siddhartth1987 
Implementation for Remove User API under Admin auth is completed

Commit:
Remove User API impl. completed

siddhartth1987 reviewed (changes requested):
The following files needn't be committed.. These are auto-generated in the target folder upon doing a clean install. 

com.upgrad.quora.api.modal.SigninResponse;
com.upgrad.quora.api.modal.SignupUserResponse;
com.upgrad.quora.api.modal.DeleteUserResponse;

Commit:
Deleted Modal files as per review comments

Swanandm1994 commented:
@siddhartth1987 Code updated with removed Modals

siddhartth1987 committed:
Merge branch 'master' into AuthAPI

siddhartth1987 committed:
Delete UserService.java

Since UserBusinessService has already been implemented this file is not required

siddhartth1987 reviewed (commented):
Removed UserService class since UserBusinessService has already been implemented
